docs/validation.md

```diff
@@ -0,0 +1,68 @@
+# Validating BuildKit CLI for kubectl
+
+While this CLI can be used for CI usecases where the images are always pushed to a registry, the primary
+usecase is for developer scenarios where the image is immediately available within the Kubernetes cluster.
+The example listed here can be used to validate everything is set up properly and you're able to build images
+and run pods with those images.
+
+## Remove any prior default builder
+```
+kubectl buildkit rm buildkit
+```
+
+## Build a simple image
+This image just emits a single line of text output and exits with success - suitable to run as a Job.
```

Comment:
Maybe:

We'll first build a simple image which prints "Success!" after it runs. We won't tag this with `latest` since that would cause Kubernetes to try to pull it from a registry.